OrbitGrpcProtos/capture.proto

```diff
@@ -128,6 +128,48 @@ message GpuJob {
   uint64 dma_fence_signaled_time_ns = 11;
 }
 
+message GpuQueueSubmission {
+  GpuQueueSubmissionMetaInfo meta_info = 1;
+  repeated GpuSubmitInfo submit_infos = 2;
+  repeated GpuDebugMarker completed_markers = 3;
+  int32 num_begin_markers = 4;
+}
+
+message GpuQueueSubmissionMetaInfo {
+  int32 tid = 1;
+  uint64 pre_submission_cpu_timestamp = 2;
+  uint64 post_submission_cpu_timestamp = 3;
+}
+
+message GpuSubmitInfo {
+  repeated GpuCommandBuffer command_buffers = 1;
+}
+
+message GpuCommandBuffer {
+  uint64 begin_gpu_timestamp_ns = 1;
+  uint64 end_gpu_timestamp_ns = 2;
+}
+
+message GpuDebugMarkerBeginInfo {
+  GpuQueueSubmissionMetaInfo meta_info = 1;
```

Comment:
It doesn't seem to be obvious why this message has a GpuQueueSubmissionMetaInfo field. Is this only for convenience and has the same data as the GpuQueueSubmission parent or is this different?

Reply:
Same here, add a comment in the code to clarify why this is needed.

Reply:
That differs, if the BeginMarkers originate from a different submission.

Reply:
Ok, that should be explained in a code comment.

Reply:
Done.